package/src/testingUtil.js

```diff
@@ -61,6 +61,9 @@ export const debuggerSetup = async (fn) => {
     consoleLogQueue.push({ method: "log", arguments: arguments });
     return _log.apply(console, arguments);
   };
+  console.log_without_reporting = function () {
```

Comment:
Retain an alias for the original console.log that we can use to output lines only to the console, and not to the user in the UI.